scripts/docstrings_checker.py

```diff
@@ -214,107 +212,3 @@ class GoogleDocstring(_check_docs_utils.GoogleDocstring):
     """.format(
         type=re_multiple_type,
     ), flags=re.X | re.S | re.M)
-
-
-class ASTDocStringChecker:
```

Comment:
Why is this deleted?

Reply:
@vojtechjelinek and I discussed that this class was not being used and If needed in the future, we can always recover it from git history, so we decided to remove it.